websites/G/G2G/presence.ts

```diff
@@ -0,0 +1,97 @@
+const presence = new Presence({
+		clientId: "1073167802296438875",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "https://i.imgur.com/XkzTOsp.png",
```

Comment:
Should use the same 512x512 logo in the metadata
```suggestion
		largeImageKey: "https://i.imgur.com/U0FQhL3.png",
```